src/models/Token/integration.spec.js

```diff
@@ -0,0 +1,59 @@
+import resetDatabase from '@tests/resetDatabase'
+import database from '@database/index'
+import { createToken, retrieveToken, validateToken } from './index'
+
+describe('The Token model helpers', () => {
+  beforeAll(async done => {
+    await resetDatabase()
+    await database.migrate.latest()
+    done()
+  })
+
+  it("should return errors if the user doesn't exist", async done => {
+    const { token, errors } = await createToken()
```

Comment:
How does this work?
You don’t pass any user to `createToken`, which is the same as passing `undefined`, but inside of `createToken` you check for `null` (`if (user === null) { … }`)?
You can keep doing that but then you should probably use `if (!user) { … }` instead.